src/sagemaker_pytorch_serving_container/torchserve.py

```diff
@@ -50,6 +50,7 @@
 
 PYTHON_PATH_ENV = "PYTHONPATH"
 REQUIREMENTS_PATH = os.path.join(code_dir, "requirements.txt")
+LOG4J_OVERRIDE_PATH = os.path.join(code_dir, "log4j.properties")
```

Comment:
We have moved to log4j2
```suggestion
LOG4J_OVERRIDE_PATH = os.path.join(code_dir, "log4j.xml")
```